lib/api/src/grpc/proto/points.proto

```diff
@@ -609,6 +615,11 @@ message DivExpression {
     optional float by_zero_default = 3;
 }
 
+message PowExpression {
+    Expression base = 1;
```

Comment:
Make this optional and default to 2?

Reply:
Why, though? I am trying to keep close to [`std`'s powf](https://doc.rust-lang.org/std/primitive.f32.html#method.powf), otherwise it would look more similar to [exp2](https://doc.rust-lang.org/std/primitive.f32.html#method.exp)